lib/BoundaryValueDiffEqShooting/src/single_shooting.jl

```diff
@@ -135,7 +140,7 @@ end
 
 function __single_shooting_jacobian_ode_cache(
         prob, jac_cache, ::DiffCacheNeeded, u0, ode_alg; kwargs...)
-    cache = jac_cache.cache
+    cache = jac_cache.config
```

Comment:
This is an implementation detail, you are not supposed to access fields of preparation results directly (remember https://github.com/JuliaDiff/DifferentiationInterface.jl/pull/672)

Reply:
Oh, yeah, my bad, forgot we can get the overloaded input